spark-native/scripts/spark/added/launch.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# spark likes to be able to lookup a username for the running UID, if
+# no name is present fake it.
+cat /etc/passwd > /tmp/passwd
+echo "$(id -u):x:$(id -u):$(id -g):dynamic uid:$SPARK_HOME:/bin/false" >> /tmp/passwd
+
+export NSS_WRAPPER_PASSWD=/tmp/passwd
+# NSS_WRAPPER_GROUP must be set for NSS_WRAPPER_PASSWD to be used
+export NSS_WRAPPER_GROUP=/etc/group
+
+export LD_PRELOAD=libnss_wrapper.so
+
+# If SPARK_MASTER_ADDRESS env varaible is not provided, start master,
+# otherwise start worker and connect to SPARK_MASTER_ADDRESS
+if [ -z ${SPARK_MASTER_ADDRESS+_} ]; then
```

Comment:
i expect there's no master w/ native k8s scheduler backend. is this condition necessary?